scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -351,6 +351,7 @@ struct layer {
 	char			name[MAX_LAYER_NAME];
 	bool			is_protected;
 	bool			periodically_refresh;
+	u8			cpuset[MAX_CPUS_U8];
```

Comment:
While it's following the existing usage, it'd probably be better to switch to u64 for bpf masks. All registers are 64bits anyway, so using u8 would just add unnecessasry masking operations.